app/views/catalog/_zero_results.html.erb

```diff
@@ -1,5 +1,9 @@
 <h1><%= t('trln_argon.search.zero_results.title', local_institution_name: institution_long_name) %></h1>
 
+<% if search_state.params.key?('f') && !search_state.params.fetch('f').empty? && @response.empty? %>
+  <h3><%= link_to t('trln_argon.search.zero_results.remove_filters'), url_for(search_state.params_for_search(f: nil)) %></h3>
```

Comment:
This is a minor point, but the story specifies that the text should be:

"<ins>Try removing your search filters</ins> to see more results." with just the underlined part linked.

You can see the pattern for how to do this in a translation starting on line 9 with the translations for `expand_search_html` and `expand_search_html_href`